packages/venia-ui/lib/components/App/app.js

```diff
@@ -83,64 +68,45 @@ const App = props => {
             if (!toasts.get(errorToastId)) {
                 addToast(errorToastProps);
             }
-        }
-    }, [errors, handleDismiss]); // eslint-disable-line
-
-    const { app, closeDrawer } = props;
-    const { drawer, hasBeenOffline, isOnline, overlay } = app;
-    const cartIsOpen = drawer === 'cart';
-
-    useEffect(() => {
-        if (hasBeenOffline) {
-            if (isOnline) {
-                addToast({
-                    type: 'info',
-                    icon: OnlineIcon,
-                    message: 'You are online.',
-                    timeout: 3000
-                });
-            } else {
-                addToast({
-                    type: 'error',
-                    icon: OfflineIcon,
-                    message:
-                        'You are offline. Some features may be unavailable.',
-                    timeout: 3000
-                });
-            }
-        }
-    }, [addToast, hasBeenOffline, isOnline]);
+        },
+        [addToast, toasts]
+    );
+
+    const talonProps = useApp({
+        handleError,
+        handleIsOffline,
+        handleIsOnline,
+        markErrorHandled,
+        renderError,
+        unhandledErrors
+    });
+
+    const { hasOverlay, handleCloseDrawer } = talonProps;
 
     if (renderError) {
         return (
-            <Fragment>
+            <HeadProvider>
+                <Title>{`Home Page - ${STORE_NAME}`}</Title>
                 <Main isMasked={true} />
                 <Mask isActive={true} />
                 <ToastContainer />
-            </Fragment>
+            </HeadProvider>
```

Comment:
I think we forgot to add this when we added `HeadProvider` and `Title` so I did so here.